Filecoin applications improvements (#466)
filecoin-project/filecoin-docs: docs/build/filecoin-pinning-services.md

---
title: 'Filecoin-backed Pinning Services'
description: "Filecoin-backed pinning services (FPS) are data storage and retrieval services that offer the performance and availability of IPFS alongside the data persistence features of Filecoin's decentralized storage network."
breadcrumb: 'FPSs architecture'
---

# {{ $frontmatter.title }}

{{ $frontmatter.description }}

::: tip
You can find a list of [existing FPS providers here](README.md).
:::

> [[TOC]]

Reviewer: ```suggestion
```

- Maintain and monitor your data storage.
- Retrieve your data through the API of the service you are using.

## Key benefits of FPSs

> 1. Data backed up by decentralized storage network (the Filecoin network) for data persistence.

Reviewer: ```suggestion
1. Data is backed up on the Filecoin network for data persistence.
```

> 2. Perfect for developers that do not want to worry about maintaining their own decentralized infrastructure while avoiding vendor lock-in. All providers contribute to the same storage network.

Reviewer: ```suggestion
2. Developers do not need to worry about maintaining their own decentralized infrastructure. Using an FPS also avoids _vendor lock-in_, since all providers contribute to the same storage network.
```

> 3. FPSs are working towards the adoption of the [IPFS Pinning Service API](https://ipfs.github.io/pinning-services-api-spec/). This common API is designed to work flexibly with any content-addressed storage solution, be it Filecoin, IPFS or services integrating both. Thus, Filecoin-specific workflows like deal-making are abstracted from the users without sacrifizing features (i.e. different geographies, redundant copies, and cryptographic storage receipts).

Reviewer: ```suggestion
3. FPSs are working towards the adoption of the [IPFS Pinning Service API](https://ipfs.github.io/pinning-services-api-spec/). This common API is designed to work flexibly with any content-addressed storage solution, be it Filecoin, IPFS, or a service integrating both. Filecoin-specific workflows are abstracted from the users without sacrificing features like different geographies, redundant copies, and cryptographic storage receipts.
```
